test/sslapitest.c

```diff
@@ -481,8 +481,53 @@ static int test_early_cb(void)
 
     return testresult;
 }
+
+static int test_no_ems(void)
```

Comment:
Doesn't this function need to set the max protocol to TLSv1.2?

Reply:
Yes.